nsis/src/nsis/file/NsisConstants.java

```diff
@@ -6,10 +6,14 @@
 			(byte) 0xde };
 	public static final byte[] NSIS_MAGIC = { 'N', 'u', 'l', 'l', 's', 'o', 'f', 't', 'I', 'n', 's',
 			't' };
-	public static final int NB_NSIS_BLOCKS = 8;
 	public static final int NSIS_MAX_STRLEN = 1024;
 	public static final int NSIS_MAX_INST_TYPES = 32;
 	public static final byte COMPRESSION_LZMA = 0x5d;
 	public static final byte COMPRESSION_BZIP2 = 0x31;
 	public static final int COMPRESSION_LZMA_HEADER_LENGTH = 5;
+
+	public enum Blocks {
+		NB_PAGES, NB_SECTIONS, NB_ENTRIES, NB_STRINGS, NB_LANGTABLES, NB_CTLCOLORS, NB_BGFONT,
+		NB_DATA, NB_BLOCKS
```

Comment:
Why prefix NB_? Previously you were using it in the sense of "number of <items>", which is not the case here.

Reply:
I used the same prefix as the spec, should I just omit the prefix?
https://sourceforge.net/p/nsis/code/HEAD/tree/NSIS/trunk/Source/exehead/fileform.h#l276

Reply:
I think you should omit the prefix, it's confusing. In the C code they had to use prefix, because C enums don't have the enum name as a "namespace" of sorts.